README.md

```diff
@@ -57,17 +61,15 @@ pyang *.yang
 ```
 Note: Refer [Pyang Errors Resolution](https://github.com/yang-infra/yangkit/tree/main#pyang-related-errors-resolution) if you face any pyang related issue.
 
-4) Create a bundle file: Example : /ws/jhanm-sjc/yang-kit/bundle.json
+4) Create a bundle file: 
+
+Example -
 ```
-(yangkit_venv) [jhanm@sjc-ads-1025 yang-kit]$ pwd
-/ws/jhanm-sjc/yang-kit
-(yangkit_venv) [jhanm@sjc-ads-1025 yang-kit]$ cat bundle.json 
  {
    "name": "cisco-ios-xr", <Bundle Name>
    "version": "7.11.1", <Image Version>
-   "yang_dir": "/ws/jhanm-sjc/yang-kit/yang_files" <Yang Files Directory>
+   "yang_dir": "/Users/jhanm/Documents/Yang_Folder/yang_files" <Absolute Path to Yang Files Directory>
```

Comment:
Shall we replace this path also to some Dummy Path in the Readme?

Something like - /Absolute_Path/Yang_File_Dir